### PR TITLE
XW-1620 | Search in global nav

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/XW-1620
https://github.com/Wikia/design-system/pull/42
Ping @Wikia/x-wing

### CODE REVIEW

File: components/global-navigation.md [hunk 1]
```diff
@@ -32,7 +32,26 @@
...
+								<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-cross"></use>
+							</svg>
+						</button>
+					</div>
+					<button type="submit" class="wds-button wds-global-navigation__search__submit">
+						<svg class="wds-icon wds-icon-small wds-global-navigation__search__submit__icon" alt="Search">
+							<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-arrow"></use>
+						</svg>
+					</button>
+				<form>
```
Comment (tomasznapieralski) [inline components/global-navigation.md]: close form? others below are also open

**Comment by tomasznapieralski** [comment]
Minor, LGTM

**Comment by Warkot** [comment]
I fixed these minor things and merged the base brach here (origin/XW-1618v2).

LGTM

File: components/global-navigation.md [hunk 2]
```diff
@@ -215,7 +273,88 @@
...
+				<svg class="wds-global-navigation__logo__fandom" alt="Fandom">
+					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-fandom"></use>
+				</svg>
+				<span class="wds-global-navigation__logo__powered-by">powered by</span>
+				<svg class="wds-global-navigation__logo__wikia" alt="Wikia">
+					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-wikia"></use>
+				</svg>
+			</a>
+			<div class="col-sm-5 col-md-6 col-lg-6 wds-global-navigation__middle-column">
+				<form class="wds-global-navigation__search wds-is-active">
```
Comment (rogatty) [inline components/global-navigation.md]: Where did the links (hubs etc.) go? We don't remove them from DOM, just hide somehow, right? I think they should be here to show that the CSS to hide them works correctly.

Comment (Warkot) [reply components/global-navigation.md]: +1 These should probably get `display: none` using a class added to a parent (`search-is-active` or sth like it) when the search is focused. This class should probably also control the look of the search itself.

**Comment by rogatty** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM

**Pull request merged**